protoc-artifacts/README.md

```diff
@@ -2,20 +2,10 @@
 ``protoc`` is the compiler for ``.proto`` files. It generates language bindings
 for the messages and/or RPC services from ``.proto`` files.
 
-Because ``protoc`` is a native executable, the scripts under this directory
-build and publish a ``protoc`` executable (a.k.a. artifact) to Maven
-repositories. The artifact can be used by build automation tools so that users
-would not need to compile and install ``protoc`` for their systems.
-
-If you would like us to publish protoc artifact for a new platform, please send
-us a pull request to add support for the new platform. You would need to change
-the following files:
-
-* [build-protoc.sh](build-protoc.sh): script to cross-build the protoc for your
-  platform.
-* [pom.xml](pom.xml): script to upload artifacts to maven.
```

Comment:
Is this one still used?  It seems weird to delete the documentation for this file but not delete the file

Reply:
This is a little in flux right now, but the pom file will only be used by us to upload the artifacts to maven so I don't think we need to document it for external users.

Reply:
Could we move it or document that it's for internal use only?  The existence of this folder seems a little confusing after this PR

Reply:
Documented that it's only for internal use. I think after the 21.x releases we may be able to restructure this folder, but I will punt on that for now.